h/views/activity.py

```diff
@@ -84,12 +84,41 @@ def group_search(request):
     if request.has_permission('admin', group):
         result['group_edit_url'] = request.route_url('group_edit', pubid=pubid)
 
+    result['more_info'] = 'more_info' in request.params
+
     return result
 
 
 @view_config(route_name='activity.group_search',
              request_method='POST',
              renderer='h:templates/activity/search.html.jinja2',
+             request_param='more_info')
+def group_search_more_info(request):
```

Comment:
I suspect quite a bit of this is going to end up duplicated when the user profile sidebar is implemented. If we can find a way to avoid too much duplication when that is implemented it would be helpful.